LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Menelaos Dimitrios Kokolios
+Copyright (c) 2020 Mauro Pelucchi
```

Comment:
Original idea code that's no longer used (iirc)  come from Menelaos so I'll like to keep copyright to him.